client/src/actions/monument.js

```diff
@@ -34,5 +80,37 @@ export default function fetchMonument(id) {
 
         if (error || res.error) dispatch(fetchMonumentError(error || res.error));
         else dispatch(fetchMonumentSuccess(res));
+
+        let queryOptions = {
+            lat: res.lat,
+            lon: res.lon,
+            d: 25,
+            limit: 6
+        };
+        let queryString = QueryString.stringify(queryOptions);
+        dispatch(fetchNearbyMonumentsPending());
+        try {
+            const nearbyMonuments = await get('/api/search/?', queryString);
+            dispatch(fetchNearbyMonumentsSuccess(nearbyMonuments));
+        } catch (error) {
+            dispatch(fetchNearbyMonumentsError(error));
+        }
+
+        const tags = res.tags.map(tag => tag.name);
+        if (tags.length > 0) {
+            queryOptions = {
+                tags: tags,
+                limit: 6
+            };
+            queryString = QueryString.stringify(queryOptions);
```

Comment:
You need to do `QueryString.stringify(queryOptions, {arrayFormat: 'comma'})` because the default array format doesn't work with spring

Reply:
Done.